src/classes/ERR_Handler_API.cls

```diff
@@ -88,9 +88,27 @@ global class ERR_Handler_API {
     * @description Creates an error record from an exception, stores it in the database (if enabled), and sends error
     *              notifications (if enabled).
     * @param e Exception to create error record from.
-    * @return context The context the exception was raised on.
+    * @param context The context the exception was raised on.
     */
     global static void processError(Exception e, String context) {
         ERR_Handler.processError(e, context);
     }
+
+    /******************************************************************************************************
+     * @description Creates error records from DML results, stores in database (if enabled) and sends error
+     *              notification (if enabled)
+     * @param dmlResult The results of a single DML operation performed using the Database class.
+     * @param records The records the DML operation was performed on.
+     * @param context The context the errors were raised on
+     */
+
+    global static void processDMLError(List<Object> dmlResult, List<SObject> records, String context) {
+
+        ERR_Handler.Errors errors = ERR_Handler.getErrors(dmlResult, records);
```

Comment:
This implementation will end up in `ERR_Handler.getErrors(List<Object> dmlResults, List<SObject> sObjects, Boolean displayErrors)` where displayErrors is true. That will cause the code to issue `ApexPages.addMessage()` if there is an error and a current apex page. Is that the intent, or could this method include a 3rd parameter of false to prevent errors from being rendered in the UI?

Reply:
Thank you @force2b! I updated the PR based on both feedback items you mentioned.